[ffmpeg] Fix cmake warning.

cbirkhold opened the pull request:
Nothing serious but unnecessary log pollution. The endif(...) statement did not match the if(...) statement. Removed all arguments from endif() as is the case for all other endif() statements in the same file and not required by cmake).

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

N/A:

- [ ] SHA512s are updated for each updated download.
- [ ] The "supports" clause reflects platforms that may be fixed by this new version.
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [ ] Any patches that are no longer applied are deleted from the port's directory.

cbirkhold committed:
[ffmpeg] Fix cmake warning.

vicroms reviewed (changes requested):
Please fix baseline issues on android triplets.

cbirkhold committed:
ffmpeg[core,alsa]:x64-android passed ...

... but ffmpeg[alsa] was marked expected to be a cascaded failure. This
fails Android CI. If this works now on Android now (unrelated to this
change) we need to update this file.
